modules/salt/pages/formula-yomi.adoc

```diff
@@ -0,0 +1,153 @@
+[[yomi-formula]]
+= Yomi Formula
+
+Yhe Yomi (yet one more installer) installer for {suse} and openSUSE operating systems is configured using formulas with forms.
+
+The ``yomi-formula`` package provides these formulas:
+
+* Yomi
+* Yomi Storage
+* Yomi Bootloader
+* Yomi Software
+* Yomi Services
+* Yomi Users
+
+
+.Procedure: Install the Yomi Formulas with Forms
+. On the {productname} Server, at the command prompt, as root, install the ``yomi-formula`` package:
++
+----
+zypper in yomi-formula
+----
+. In the {susemgr} {webui}, open the details page for the Salt master, and navigate to the Formulas tab.
+. Check the [guimenu]``Installer`` box, and click btn:[Save].
+. Apply the highstate.
```

Comment:
hrm, I don't want to repeat all that, since it's already in the Yomi chapter. I've added a bit of a summary and an xref instead. Let me know if that's better.